### PR TITLE
Add browse inline search with pagination

### PR DESCRIPTION
**Pull request by pgobin**
- Add basic browse search
- Support search pagination

Signed-off-by: PGobz <p.gobin@gmail.com>

### CODE REVIEW

File: client/src/sagas/SearchSaga.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { searchBrowse } from 'sagas/SearchSaga';
+
+
+const stepper = (fn) => (mock) => fn.next(mock).value;
+
+
+test('browse: success path', () => {
+  const query = {
+    query: {
+      search_input: { value: 'a' },
```
Comment (nadiabahrami) [inline client/src/sagas/SearchSaga.test.js]: This needs to change

File: client/src/sagas/SearchSaga.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  expect(stepRes).toEqual(
+    put(SearchActions.searchBrowseSuccess(res.data.data))
+  );
+});
+
+
+test('browse: failure path', () => {
+  const query = {
+    query: {
+      search_input: { value: 'a' },
```
Comment (nadiabahrami) [inline client/src/sagas/SearchSaga.test.js]: Same here

File: client/src/sagas/SearchSaga.test.js (hunk 1 above)
Comment (pgobin) [reply client/src/sagas/SearchSaga.test.js]: Fixed!

File: client/src/sagas/SearchSaga.test.js (hunk 2 above)
Comment (pgobin) [reply client/src/sagas/SearchSaga.test.js]: Fixed!

**Pull request merged**